cabal-install/tests/IntegrationTests/exec/should_run/exit_with_failure_without_args.sh

```diff
@@ -0,0 +1,6 @@
+. ../common.sh
+
+# We should probably be using a .err file and should_fail,
```

Comment:
One possible future solution for these kinds of problems is to add support for platform-specific `.err` files.